web/src/compose_recipient.ts

```diff
@@ -83,12 +85,40 @@ export function update_narrow_to_recipient_visibility(): void {
             compose_state.has_full_recipient()
         ) {
             $(".conversation-arrow").toggleClass("narrow_to_compose_recipients", true);
+            show_tippy_tooltip();
             return;
         }
     }
     $(".conversation-arrow").toggleClass("narrow_to_compose_recipients", false);
 }
 
+function show_tippy_tooltip(): void {
+    const arrow_element = document.querySelector(".conversation-arrow");
+    if (!arrow_element) {
+        return;
+    }
+    if (localStorage.getItem("conversation_tooltip")) {
+        return;
+    }
+
+    const instance = tippy(arrow_element, {
+        content: "Go to conversation",
+        trigger: "manual",
+        placement: "top",
+    });
+
+    instance.show();
+    localStorage.setItem("conversation_tooltip", "true");
```

Comment:
Don't use browser local storage, see `git grep OnboardingStep` for how to implement a one-time notice.

Reply:
Im fixing it right now.